.gitignore

```diff
@@ -2,6 +2,5 @@
 .idea/
 bin/
 npm-debug.log
-dist/
```

Comment:
is this on purpose? :)

Reply:
also all the files added from `dist/` as part the effect of this can be removed I think?

Reply:
@karniwl I added the `dist/` files because I was referencing the snyk-docker-plugin as a git branch inside package.json of kubernetes-monitor 😄For that to work there need to be some published/built assets in order to build our kubernetes-monitor container (I don't know why but that's the reality... npm install locally works but when inside the container/Dockerfile it just fails with missing files)